### PR TITLE
Updated Hospital View

### PR DESCRIPTION
**Pull request by cristomathew**
A better UI for hospital View

### CODE REVIEW

**Comment by MansoorMajeed** [comment]
@cristomathew I think it would be nicer if we added a screenshot of the proposed change?

File: src/Components/Common/HospitalView.tsx [hunk 1]
```diff
@@ -5,12 +5,17 @@ import CardActions from '@material-ui/core/CardActions';
...
+import PhoneIcon from '@material-ui/icons/Phone';
+import LocationCityIcon from '@material-ui/icons/LocationCity';
+import LocationOnIcon from '@material-ui/icons/LocationOn';
+import LocalHospitalIcon from '@material-ui/icons/LocalHospital';
 const useStyles = makeStyles({
     root: {
       minWidth: 200,
-      width: 300,
+      width: 350,
+      backgroundImage: "linear-gradient(to right top, #0a3ded, #0053f5, #0066fa, #0078fd, #0088ff, #0099ff, #00a9ff, #00b8ff, #00cbff, #00dcff, #25ecf9, #5ffbf1)",
```
Comment (dauntlessnomad) [inline src/Components/Common/HospitalView.tsx]: fix the color to a lighter one

**Comment by cristomathew** [comment]
Screenshot 2020-03-24 at 11 17 34 AM
latest colour change

**Comment by dauntlessnomad** [comment]
@cristo remove the blue . We are following extreme light colours

**Comment by cristomathew** [comment]
Screenshot 2020-03-24 at 10 04 57 PM
@bodhish can you review this

**Pull request merged**